remediation/fence-agents-remediation/configuration.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Configuration
+parent: Fence Agents Remediation
+grand_parent: Remediation
+nav_order: 2
+---
+
+## Fence Agents Remediation Configuration
+
+Fence Agents Remediation (FAR) uses a Custom Resource (CR) created by [Node Health Check Operator](https://github.com/medik8s/node-healthcheck-operator#readme) (NHC) based on a specific remediation template (*FenceAgentsRemediationTemplate*).
```

Comment:
why "specific" remediation template?

Reply:
I was trying to show the importance of using this template when using FAR with NHC. So that using any other template for FAR and NHC won't work. Is that kind of trivial that we should avoid of expressing or the word `specific` just does not fit here?

Reply:
IMHO using a template is trivial, it's how NHC works in gerneral